src/app/components/map/map.component.ts

```diff
@@ -83,18 +84,29 @@ export class MapComponent implements OnInit, OnChanges {
   }
 
   initMap(): void {
-    console.log('Calling initMap');
-
-    console.log('Array before API call:');
-    console.log(this.machines);
-
     // Get map data, put coordinates into machines array
     this.vmMapsQuery.getById(this.mapIdInput).subscribe((data) => {
       this.name = data.name;
       this.mapId = data.id;
-      this.imageURL = data.imageUrl;
       this.teamIDs = data.teamIds;
 
+      // As in display component, need to figure out if the URL is b64 encoded and, if so, convert it to a blob
+      // and then generate an object url
+      const url = data.imageUrl;
+      console.log('URL = ' + url);
```

Comment:
lots of console.logs here, how much do we care about clean consoles?

Reply:
Yea I would prefer to remove the log statements once dev is finished I think.